packages/core/src/common/menu.ts

```diff
@@ -228,10 +233,17 @@ export class CompositeMenuNode implements MenuNode {
 }
 
 export class ActionMenuNode implements MenuNode {
+
+    readonly altNode: ActionMenuNode;
```

Comment:
For the sake of type-safety, wouldn't it make sense to have `readonly altNode: ActionMenuNode | undefined;` instead?